frontend/src/components/applicants.tsx

```diff
@@ -47,6 +47,7 @@ export function ApplicantsList(props: {
             columns={columns}
             showPagination={false}
             defaultPageSize={pageSize}
+            pageSize={pageSize}
```

Comment:
change `defaultPageSize={pageSize}` to `defaultPageSize={Number.MAX_SAFE_INTEGER}`
and remove `pageSize={pageSize}` and the declaration of `pageSize` above.

`Infinity` doesn't work, but `Number.MAX_SAFE_INTEGER` has the same effect